en/advanced_config/pid_tuning_guide_multicopter.md

```diff
@@ -1,74 +1,206 @@
-There are substantially better performing control techniques (LQR/LQG) from the model predictive control (MPC), 
-but since these techniques require a more or less accurate model of the system, they are not as widely used. 
-The goal of all PX4 control infrastructure is to move as soon as possible on MPC, 
-but since not all supported systems models are available, 
-PID tuning is very relevant (and PID control is sufficient for many cases).
+PX4 uses **P**roportional, **I**ntegral, **D**erivative (PID) controllers, which are the most widespread control technique. 
+
+The controllers are layered, which means a higher-level controller passes its
+results to a lower-level controller. The lowest-level controller is the the **rate
+controller**, then there is the **attitude contoller**, and then the **position
+controller**.
+The PID tuning needs to be done in the same order, starting with the rate
+controller, as it will affect all other controllers.
+
 
 ## Precondition
+- You have chosen the closest matching airframe. It should give you a vehicle
+  that already flies.
+- You should have done an [ESC calibration](esc_calibration.md).
+- `PWM_MIN` is set correctly. It needs to be set low, but such that the **motors
+  never stop** when the vehicle is armed. This can be tested in acro
+  or in manual mode, without props:
+  - arm the vehicle and lower the throttle to the minimum,
+  - tilt the vehicle to all directions, about 60 degrees,
+  - check that no motor turns off.
+- Optionally enable the high-rate logging profile with the `SDLOG_PROFILE`
+  parameter. It will help evaluating the rate and attitude tracking performance
+  from the log. The option can be disabled afterwards.
+
+> **Warning** Always disable `MC_AIRMODE` when tuning a vehile. 
 
-The PID-Gains should be chosen such that tracking is as tight as possible. Before doing any position/velocity control related tuning,
-turn off all [higher-level tuning](advanced_mc_position_tuning.md).
 
-- [MPC_ACC_HOR_MAX](parameter_reference.md#MPC_ACC_HOR_MAX): 1000
-- [MPC_ACC_HOR](parameter_reference.md#MPC_ACC_HOR) : 1000
-- [MPC_DEC_HOR_SLOW](parameter_reference.md#MPC_DEC_HOR_SLOW) : 1000 
-- [MPC_ACC_UP_MAX](parameter_reference.md#MPC_ACC_UP_MAX) : 1000 
-- [MPC_ACC_DOWN_MAX](parameter_reference.md#MPC_ACC_DOWN_MAX) : 1000 
-- [MPC_JERK_MAX](parameter_reference.md#MPC_JERK_MAX) : 0 
-- [MPC_JERK_MIN](parameter_reference.md#MPC_JERK_MIN) : 1
 
+## Tuning steps
+
+> **Note** For safety reasons, the default gains are set to a small value.  
+> You must increase the gains before you can expect good control responses. 
+
+Here are some general points to follow when tuning:
+- All gains should be increased very slowly, by 20-30% per iteration, and
+  even 5-10% for final fine tuning.
+  Note that too large gains may cause dangerous oscillations!
+- It is advised to always quickly land when changing a parameter, and then

```

Comment:
I think you are saying that all parameters should be changed when landed in order to check for oscillations? Will everyone know what you mean by oscillations - wobbling, rotating?